### PR TITLE
docs: add asterisk to 'use' for more clarity

### PR DESCRIPTION
**Pull request by khalilou88**
# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Use cases and why

- 1. One use case
- 2. Another use case

# Checklist:

- [ ] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/docs/src/routes/docs/(qwik)/components/overview/index.mdx [hunk 1]
```diff
@@ -286,7 +286,7 @@ You can think of inline components as being inlined into the component where the
 ### Limitations
 Inline components come with some limitations that the standard `component$()`
 does not have. Inline components:
-- Cannot use `use` methods such as `useSignal` or `useStore`.
+- Cannot `use` methods such as `useSignal` or `useStore`.
```
Comment (zanettin) [inline packages/docs/src/routes/docs/(qwik)/components/overview/index.mdx:289]: imo the word `use` is correct. the 2nd `use` stands for all hooks like `useVisibleTask`, `useOn`. guess the best update here would be smth like this:

```suggestion
- Cannot use `use*` methods such as `useSignal` or `useStore`.
```

**Review by zanettin — COMMENTED** [review]
Hi @khalilou88 👋 
Thanks for creating this PR 🎉  just have a tiny input 😄

**Comment by khalilou88** [comment]
@zanettin Thanks for your comment. I modified the PR

**Comment by zanettin** [comment]
thx again 🙏 and welcome in the contributor family 😃

**Pull request merged**